carball/analysis/analysis_manager.py

```diff
@@ -74,44 +95,64 @@ def perform_full_analysis(self, game: Game, proto_game: game_pb2.Game, player_ma
                               calculate_intensive_events: bool = False):
 
         """
-        Performs the more in depth analysis on the game in addition to just metadata.
-        :param game: Contains all metadata about the game and any json data.
-        :param proto_game: The protobuf where all the stats are being stored.
+        Sets some further data and cleans the replay;
+        Then, performs the analysis, which includes:
+            creating in-game event data (boostpads, hits, carries, bumps etc.)
```

Comment:
Try not to give too much info about what the internals do.

Those can change which then means the doc is out of date.
People using it do not necessarily need to know explicitly what it does because they can get that by reading the code.  But noting any side effects is useful

Reply:
Cool, will take note